uninstall.php

```diff
@@ -35,8 +38,11 @@
 	$give_post_types = [ 'give_forms', 'give_payment' ];
 	foreach ( $give_post_types as $post_type ) {
 
-		$give_taxonomies = array_merge( $give_taxonomies, get_object_taxonomies( $post_type ) );
-		$items           = get_posts(
+		foreach ( get_object_taxonomies( $post_type ) as $taxonomy ) {
```

Comment:
Is the loop necessary since we are just assigning the values to a new array? How does the loop improve the `array_merge`?